decred_wallet/Extensions/Extensions.swift

```diff
@@ -69,3 +69,12 @@ extension Date {
         self = Date(timeIntervalSince1970: TimeInterval(milliseconds) / 1000)
     }
 }
+
+extension UIRefreshControl {
+    //display loading indicator without the aid of user swipping down tableview
```

Comment:
Some other dev seeing this code might be tempted to fix it, it doesn't seem like the most logical or straightforward way to display a loading indicator. Your comment should include why you're using this approach and probably other approaches you considered and why they would not work. May also include the link to SO comment/answer that guided this your implementation.

See https://medium.freecodecamp.org/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83

Reply:
UIRefreshControl has its loading indicator which is used for tableview swipe down refresh, reusing  same loader in this case seems straightforward.